mobile skeleton for distribution module

durrello opened the pull request:
please if you can merge this so when my group members are puling they can have this
- created the screens for this module
- linked the screens as it will in final stage
- no attention payed to design
- still has to refactor code
- added some drawables
- added some strings
- added some asserts

durrello committed:
mobile skeleton for distribution module
- created the screens for this module
- linked the screens as it will in final stage
- no attention payed to design
- added some drawables
- added some strings
- added some asserts

xurror reviewed (changes requested):
Please can you remove all the .idea/** files include here. Maybe add that a .gitignore so it's not included again in the future?

durrello committed:
few updates on the distribution module, changes made either on the .xml file, java file or both
- added google maps to get user location screen from click of button
- added google maps so agent can track user's location, dmDisTrack screen
- coded the buttons for onClick
- Substituted TextView to EditText, dmHealthInfo
- refactored some code

durrello committed:
Added .idea folder and files to .gitignore

durrello committed:
Remove .idea folder from repo

durrello commented:
sry didn't read this message earlier before second commit but did this right after

xurror commented:
No problem. I'll just blindly merge this.
